product_docs/docs/epas/14/epas_platform_support/index.mdx

```diff
@@ -34,9 +34,9 @@ Additional disk space is required for data or supporting components.
 
 **User Privileges**
 
-To perform an Advanced Server installation on a Windows system you must have administrator privilege. If you are installing Advanced Server on a Windows system that is configured with `User Account Control` enabled, you can assume sufficient privileges to invoke the graphical installer by right clicking on the name of the installer and selecting `Run as administrator` from the context menu.
+To perform an EDB Advanced Server installation on a Linux system you must have superuser or administrator or sudo privileges.
```

Comment:
use either the full name (EDB Postgres Advanced Server) or short name (Advanced Server). Best practice, if using the short name) is to introduce it in the first instance. For example, EDB Postgres Advanced Server (Advanced Server)

Reply:
Done the changes